ipaplatform/redhat/authconfig.py

```diff
@@ -95,18 +94,13 @@ def _parse_authselect_output(self, output_text=None):
             if output_text is None:
                 return None
 
-        cfg_params = re.findall(
-            r"\s*Profile ID:\s*(\S+)\s*\n\s*Enabled features:\s*(.*)",
-            output_text,
-            re.DOTALL
-        )
-
-        profile = cfg_params[0][0]
+        output_items = output_text.split(' ')
```

Comment:
With this code, the last item (either feature or profile) will contain a newline:

```
>>> from ipapython import ipautil
>>> from ipaplatform.paths import paths
>>> current = ipautil.run([paths.AUTHSELECT, "current", "--raw"])
>>> output_text = current.raw_output.decode()
>>> output_text.split(' ')
['sssd\n']
```

I suggest to modify the code to:

```
output_text = output_text.strip()
if not output_text:
    return None
output_items = output_text.split(' ')
profile = output_items[0]
features = output_items[1:]
return profile, features
```